ADAL/src/request/ADAcquireTokenSilentHandler.m

```diff
@@ -301,7 +301,7 @@ - (void)getAccessToken:(ADAuthenticationCallback)completionBlock
     }
     
     // If we have a good (non-expired) access token then return it right away
-    if (item.accessToken && !item.isExpired)
+    if (item.accessToken && !item.isExpired && !_requestParams.forceRefresh)
```

Comment:
Do we even need to lookup access tokens if force refresh == true?

Reply:
discussed offline, we may need it as single resource RT lookup needs it.